ci-operator/config/opendatahub-io/vllm/opendatahub-io-vllm-ibm_main.yaml

```diff
@@ -29,7 +29,7 @@ images:
     value: 23.11.0-0
   context_dir: .
   dockerfile_path: Dockerfile.ubi
-  to: vllm-main-cuda-intel-gpu
+  to: vlm-build-cuda-amd64
```

Comment:
Will want to reference the branch in this file, openshift-ci gets confused if multiple images share a name